src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -110,6 +111,7 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
         .withPoolingOptions(poolingOptions)
         .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
         .withQueryOptions(new QueryOptions().setFetchSize(fetchSize))
+        .withPort(port)
```

Comment:
Right, and more specifically, if the client needs to contact hosts that are not in the list of contact points, this is the only way to specify the port as far as I can tell.